build.gradle

```diff
@@ -72,6 +72,7 @@ project('common') {
 }
 
 def withoutLog4j = { exclude group: 'org.slf4j', module: 'slf4j-log4j12' }
+def withoutSlf4jSimple = { exclude group: 'org.slf4j', module: 'slf4j-simple' }
```

Comment:
I think this can probably be combined with the above, and can be called withoutLogger. As that's essentially what we are overriding.

Reply:
done.